lib/yggdrasil/player/registry.ex

```diff
@@ -0,0 +1,46 @@
+defmodule Yggdrasil.Player.Registry do
+  use GenServer
+  require IEx
+
+  @registry_ets :player_registry_ets
```

Comment:
Well I was thinking this would keep track of all active players and what would expand is the data it stored (maybe) or how it's stored. Using a named table means that the `is_online/1` function can perform the lookup without having to send a message to the process.